will/requirements/base.txt

```diff
@@ -1,28 +1,29 @@
 APScheduler==2.1.2
 beautifulsoup4==4.6.0
-bottle==0.12.7
+bottle==0.12.19
 CherryPy==3.6.0
-clint==0.3.7
-dill==0.2.1
+clint==0.5.1
+dill==0.3.3
 dnspython==1.15.0
 fuzzywuzzy==0.15.1
 Jinja2==2.7.3
-Markdown==2.3.1
+Markdown==3.3.4
 MarkupSafe==0.23
 # Temporary fork of natural, until python 3 support is merged: https://github.com/tehmaze/natural/pull/13
+# Temporary fork of natural, until python 3 support is merged: https://github.com/tehmaze/natural/pull/13
 # natural==0.2.1
 will-natural==0.2.1.1
 parsedatetime==1.1.2
-python-Levenshtein==0.12.0
+python-Levenshtein==0.12.1
 pyasn1-modules==0.0.5
 pyasn1==0.1.7
 pycrypto==2.6.1
 pygerduty==0.28
-pytz==2017.2
-PyYAML==3.13
+pytz==2021.1
+PyYAML==5.4.1
 regex==2017.9.23
 redis==2.10.6
-requests==2.20.0
-six==1.10.0
-urllib3==1.24.3
-websocket-client==0.44.0
+requests==2.25.0
+six==1.15.0
+urllib3==1.25.10
+aiohttp
```

Comment:
Can you remove this since it's slack specific and is already in that requirement file?